.github/workflows/tests.yml

```diff
@@ -14,50 +15,53 @@ jobs:
       AWS_REGION: ${{ secrets.AWS_REGION }}
       EC2_SECRET: ${{ secrets.EC2_PWD }}
       S3_BACKEND_KEY_NAME: tf-state-${{ github.sha }}
+    outputs:
+      runner: ${{ steps.determine-runner.outputs.runner }}
     steps:
+      - id: determine-runner
+        name: Determine runner
+        run: if [ -z $AWS_ACCESS_KEY_ID ]; then echo "::set-output name=runner::ubuntu-latest"; else echo "::set-output name=runner::${{ github.sha }}"; fi;
       - name: checkout code
+        if: ${{ env.AWS_ACCESS_KEY_ID }}
         uses: actions/checkout@v2.1.0
       - name: setup node
+        if: ${{ env.AWS_ACCESS_KEY_ID }}
```

Comment:
Can't we move this one level higher so that it's not repeated? It should be possible according to https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
Example here https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions

Reply:
i have already tried that, but i wasn;t accepted as valid syntax https://github.com/galacticcouncil/HydraDX-node/actions/runs/793368736

Reply:
Let's try the owner method and then LGTM? We can get rid of all these ifs.

Reply:
repository owner is no/go, it is the one that fork is pull request against, so always galacticcouncil... I'm trying some other variable that could work